src/eval/methods.rs

```diff
@@ -185,6 +186,38 @@ pub fn call(
                     }
                     _ => return missing(),
                 }
+            } else if let Some(&datetime) = dynamic.downcast::<Datetime>() {
+                match method {
+                    "display" => {
+                        let pattern = args.eat()?;
+                        match datetime.display(pattern) {
+                            Ok(d) => Value::Str(Str::from(d)),
+                            Err(msg) => bail!(args.span, msg),
+                        }
+                    }
```

Comment:
```suggestion
                    "display" => datetime.display(args.eat()?).at(span)?.into(),
```

Reply:
I changed this, but did `.at(args.span)` instead of `.at(span)` since it makes more sense to me to highlight the args part here. Is that okay?